tuxemon/core/item.py

```diff
@@ -187,6 +222,35 @@ class with the name of the effect you define in item.db.
 
         return meta_result
 
+    def learn(self, user, target, slug):
+        """This effect teaches the target the technique in the tech string.
+
+        :param user: The monster or object that is using this item.
+        :param target: The monster or object that we are using this item on.
+        :param tech: The slug of the technique to teach.
+
+        :type user: Player
+        :type target: Monster
+        :type tech: String
+
+        :rtype: bool
+        :returns: Success
+
+        **Examples:**
+        >>> tm = Item("tm_blossom")
+        >>> tm.learn(player1, bulbatux, "blossom")
+        """
+        
+        try:
+            tech = Technique()
+            tech.load(slug)
+            target.learn(tech)            
+        except RuntimeError:
```

Comment:
when is a RuntimeError raised?  What circumstances does it happen? I would rather allow that exception to bubble up than to cover it here.

Reply:
please respond before resolving discussions

Reply:
Saw no need to respond, I just made the change you requested.